source/guides/guides/test-retries.md

```diff
@@ -0,0 +1,185 @@
+
+- `runMode` allows you to define the number of test retries when running `cypress run`
+- `openMode` allows you to define the number of test retries when running `cypress open`
+
+```jsx
+{
+  "retries": {
+    // Configure retries for `cypress run` 
+    // Default is 2
+    "runMode": 1,
+    // Configure retries for `cypress open`
+    // Default is 0
+    "openMode": 3
+  }
+}
+```
+
+### Custom Configurations
+
+#### Individual Test(s)
+
+In the event a specific test requires a custom number of test retries configured, this can be achieved by using the test's local configuration option.
+
+```jsx
+// Customizing retries for an individual test
+describe('User sign-up and login', () => {
+  // `it` test block with no custom configuration
+  it('should redirect unauthenticated user to sign-in page', () => {
+    ...
+  })
+
+  // `it` test block with custom configuration
+  it('allows user to login', { 
+    retries: {
+      runMode: 3,
+      openMode: 2
+    }
+  }, () => {
+    ...
+  })
+})
+
+```
+
+#### Test Suite(s)
+
+If you want a suite of tests to re-run in requires a custom number of test retries configured, this can be achieved by using the test's local configuration option
+
+```jsx
+// Customizing retries for a suite of tests
+describe('User bank accounts', {
+  retries: {
+    runMode: 3,
+    openMode: 1,
+  }
+}, () => {
+  // Individual tests will be run normally
+  it('allows a user to view their transactions, () => {
+    ...
+  }
+
+  it('allows a user to edit their transactions, () => {
+    ...
+  }
+})
+
+```
+
+You can find more information about custom configurations here: {% url "Test Configuration" configuration#Test-Configuration %}
+
+## Assets
+
+With test retries, Cypress will now generate any assets (i.e., screenshots, video recordings, etc.) per each test attempt. The assets will be properly labeled with the attempt number appended at the end of each asset.
+
+    🖼 Insert screenshot
+
+## Tips and Strategies
+
+While test retries are great for helping to avoid false negatives from failing an entire test run, it is not a good replacement for writing good tests. As a result, here are some tips and strategies to keep in mind in order to maximize the effectiveness of your tests with test retries:
+
+- If you are noticing that you need to increase the number of retries, this cause is more likely due to how the tests are written and is worth spending the time to investigate
+- If you use `Run all specs` a lot in `cypress open` mode, make sure to configure your Cypress instance to have a global `retries` of `2` so you can simulate what is being run in `cypress run` mode.
+
+## Frequently Asked Questions (FAQs)
+
+#### Will runs with test retries be counted as more than one run?
+
+No. When recording to the Cypress Dashboard, runs with test retries will be counted as a single run.
```

Comment:
We don't charge per **run**, we charge per **test**. So, a single `it` definition is counted in your billing. This is VERY important to get the right terminology here as people are confused very often. Just avoid the use of the term 'run' altogether when discussing billing. 

```suggestion
#### Will tests that retry be counted as more than one test recording in my billing?

No. Tests recorded during `cypress run` with the `--record` flag will be counted the same with or without test retries.

We consider each time the `it()` function is called to be a single test for billing purposes. The test retrying will not count as extra test recordings in your billing.

You can always see how many tests you’ve recorded from your organization’s Billing & Usage page within the {% url "Dashboard" https://on.cypress.io/dashboard %}.
```

Reply:
@jennifer-shehane Thanks for clarifying the terminology. I had heard "run" used a couple of places and thought it was the more generic / appropriate term, but now I know better! Appreciate it 🙏